api/src/application/client/client-service.js

```diff
@@ -97,6 +99,29 @@ module.exports = (bookshelf) => ({
     return bookshelf.model('client').forge({ client_id: id }).save(toStore);
   },
 
+  createResetPasswordTemplate(template, clientId) {
```

Comment:
Please create a `templates/template-service.js` file with a function called `createTemplate` that just takes a template string and returns a promise that resolves with the new template object.

Rename the `createResetPasswordTemplate` function to be like `setResetPasswordTemplate(templateId, clientId)` and update the logic to no longer have to create the template record